test/cppsuite/configs/hs_cleanup_default.txt

```diff
@@ -18,7 +18,8 @@ runtime_monitor=
     (
         enabled=true,
         limit=100
-    )
+    ),
+    postrun_statistics=[cache_hs_insert:50000:300000, cc_pages_removed:1000:8000]
```

Comment:
Could you explain in a comment in the config what the rough value was when you tested? These are quite large ranges, if we can tighten them up a bit it'll have more value. Obviously too tight and then it'll fail with a false positive.

Reply:
I'm concerned that the Evergreen machines are going to give significantly different numbers than my local machine. I'll see what values I get on those boxes.

By the way, are these the stats you're interested in?

Reply:
They are relevant to this workload so yeah it is. The comment about evergreen machines is a fair one and can wait till WT-7762 is completed.

Reply:
So no changes required here. 👍

Reply:
3905e0e